layouts/partials/news/main_prefix.html

```diff
@@ -0,0 +1,6 @@
+<main>
```

Comment:
I do have some concerns with us overriding this file.  This new section will miss out on future updates from the base theme.

The default shortcode allows authors to disable the title, can we leverage that?
https://github.com/EclipseFdn/hugo-solstice-theme/blob/master/layouts/partials/main_prefix.html

We can define an Archetypes https://gohugo.io/content-management/archetypes/ for the default values we expect authors to use.

If not, we can merge this as-is but my preference would be to use our default file to make sure that we can pull-in future enhancement from our base theme.

Reply:
The issue with main_prefix is I need hide_page_title for taxonomy and list pages, for which I need to override current functionality.

As for archetypes will create default `archetypes/news.md` have not found any currently used but I'm glad to be first one.

Reply:
I would expect the site to use our defaults if we don't have one in the Jakarta.ee repository:
https://github.com/EclipseFdn/hugo-solstice-theme/tree/master/archetypes

Reply:
it cannot we need categories/tags which it doesn't have :)

Reply:
Provided simple archetype with main fields expected from an article, as to not confuse authors.

We can provide more fields or that link in README...